### PR TITLE
PD-JS3-W3

### PR DESCRIPTION
**Pull request by esma-g**
## What does this change?

Module: JS3
Week(s): 3

## Checklist

- [x] I have read the [contributing guidelines](https://syllabus.codeyourfuture.io/contributing/overview)
- [x] I have checked my spelling and grammar with an [automated tool](https://www.grammarly.com/grammar-check)
- [x] I have previewed my changes to check the [markdown renders](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) as I intend
- [x] I have run my code to check it works
- [x] My changes follow our [Style Guide](https://syllabus.codeyourfuture.io/guides/code-style-guide)

## Description

Transferred PD content from [the PD website](https://personal-development.codeyourfuture.io/em-modules/javascript-3/week-3-agile-scrum-planning)

## Who needs to know about this?

@kfklein15 @ElizabethlawalCYF @CodeYourFuture/syllabus-team 

Rendered Pages
https://syllabus.codeyourfuture.io/js-core-3/week-3/lesson
https://syllabus.codeyourfuture.io/js-core-3/week-2/homework

### CODE REVIEW

File: docs/js-core-3/week-3/lesson.md [hunk 1]
```diff
@@ -538,6 +539,47 @@ The project is called [TV Show](../tv-show-dom-project/readme.md)
...
+
+**Exercise objective:** To use Gitprojects for planning
+
+In small groups, plan your TV show project.
+
+Step 1: Click on the projects tab in your repository and select create a project.
+
+Step 2: Name your project TV-Show-Project-Planning
+
+Step 3: Work together and review all of the tasks you need to do to complete your individual projects (please note that you might be at different stages of completion). Organise your tasks in four categories (Backlogged, To do, In progress, Done).
```
Comment (kfklein15) [inline docs/js-core-3/week-3/lesson.md]: Backlog, not Backlogged :)

**Review by kfklein15 — CHANGES_REQUESTED** [review]
one tiny change

File: docs/js-core-3/week-3/lesson.md (hunk 1 above)
Comment (esma-g) [reply docs/js-core-3/week-3/lesson.md]: > Backlog, not Backlogged :)

Done

**Review by kfklein15 — APPROVED** [review]
lgtm

**Review by illicitonion — APPROVED** [review]

**Pull request merged**